### PR TITLE
Refactored the duplicate code into a function in _ConvNd

### PR DESCRIPTION
**Pull request by noelmat**
Fixes #{issue number}

### CODE REVIEW

**Review by jerryzh168 — APPROVED** [review]
Looks good, thanks!

File: torch/nn/quantized/modules/conv.py [hunk 1]
```diff
@@ -236,22 +256,10 @@ def from_float(cls, mod):
...
-        qconv = cls(mod.in_channels, mod.out_channels, mod.kernel_size,
-                    mod.stride, mod.padding, mod.dilation, mod.groups,
-                    mod.bias is not None, mod.padding_mode)
-        qconv.set_weight_bias(qweight, mod.bias)
-        qconv.scale = float(act_scale)
-        qconv.zero_point = int(act_zp)
-
-        return qconv
-
+
```
Comment (jerryzh168) [inline torch/nn/quantized/modules/conv.py:259]: got some warnings in internal tools saying blank lines contains white space, could you remove the whitespace? (https://www.flake8rules.com/rules/W293.html)

File: torch/nn/quantized/modules/conv.py [hunk 2]
```diff
@@ -236,22 +256,10 @@ def from_float(cls, mod):
...
-        qconv.set_weight_bias(qweight, mod.bias)
-        qconv.scale = float(act_scale)
-        qconv.zero_point = int(act_zp)
-
-        return qconv
-
+        
+        return cls.get_qconv(mod, activation_post_process)
 
+
```
Comment (jerryzh168) [inline torch/nn/quantized/modules/conv.py:262]: same here I think, please check for all blank lines in the change.

File: torch/nn/quantized/modules/conv.py (hunk 1 above)
Comment (noelmat) [reply torch/nn/quantized/modules/conv.py:259]: Yes. I will do that. Do I submit a new pull request after that?

**Pull request closed without merging**